hack/test-integration.sh

```diff
@@ -69,6 +69,7 @@ os::log::start_system_logger
 function exectest() {
 	echo "Running $1..."
 
+	export TEST_ETCD_DIR=$(mktemp ${TMPDIR:-/tmp}/etcd-XXXXXXXX)
```

Comment:
If we're going for "predictable" we  want to be predictable from yet another layer -- Jenkins. Don't use `mktemp`, just name it `"${TMPDIR:-/tmp}/etcd-data"` or whatever